src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -43,6 +46,20 @@ def parse_environment_variable(value):
         return value
 
 
+def load_inputs_from_sample(sample: Union[dict, str, PathLike]):
+    if not sample:
+        return {}
+    elif isinstance(sample, dict):
+        return sample
+    elif isinstance(sample, (str, Path)) and Path(sample).suffix.lower() == ".json":
+        if not Path(sample).exists():
+            raise InvalidSampleError(f"Cannot find sample file {sample}.")
+        with open(sample, "r") as f:
+            return json.load(f)
+    else:
+        raise InvalidSampleError("Only dict and json file are supported as sample in prompty.")
```

Comment:
Test this error

Reply:
There is a test case for this negative scenario.